lib/Router.php

```diff
@@ -266,19 +266,19 @@ public function route(string $method, string $uri, ...$actions): Router {
         }
 
         $actions = array_merge($this->actions, $actions);
+        $methods = is_array($methods) ? $methods : [$methods];
```

Comment:
Just cast to array here, i.e. in the foreach: `foreach ((array) $methods as $method)`